cmd/antrea-agent/config.go

```diff
@@ -101,10 +101,16 @@ type AgentConfig struct {
 	// Enable metrics exposure via Prometheus. Initializes Prometheus metrics listener
 	// Defaults to true.
 	EnablePrometheusMetrics bool `yaml:"enablePrometheusMetrics,omitempty"`
-	// Provide the flow collector address as string with format <IP>:<port>[:<proto>], where proto is tcp or udp. This also
-	// enables the flow exporter that sends IPFIX flow records of conntrack flows on OVS bridge. If no L4 transport proto
-	// is given, we consider tcp as default.
-	// Defaults to "".
+	// FlowCollectorAddr is the input address to either connect to the IPFIX Flow Aggregator
+	// service or an external IPFIX Flow Collector.
+	// Provide the input address as "flow-aggregator.flow-aggregator.svc[:<proto>]"
+	// to connect to the Flow Aggregator over a given L4 transport protocol.
+	// Provide the input address as string with format <IP>:<port>[:<proto>] to
+	// connect to an external Flow Collector.
+	// We support tcp and udp L4 transport protocols. If no proto is given, we consider
+	// tcp as default. IP can be either IPv4 or IPv6. However, IPv6 address should
+	// be wrapped with [].
+	// Defaults to "flow-aggregator.flow-aggregator.svc".
```

Comment:
Based on the rest of the code and the yaml manifests, it seems the default is actually `"flow-aggregator.flow-aggregator.svc:tcp"`?